FPGA/CPU_GARAGE.qsf

```diff
@@ -187,8 +187,8 @@ set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to HEX5[4]
 set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to HEX5[5]
 set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to HEX5[6]
 set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to HEX5[7]
-set_instance_assignment -name IO_STANDARD "3.3 V SCHMITT TRIGGER" -to BUTTON[0]
-set_instance_assignment -name IO_STANDARD "3.3 V SCHMITT TRIGGER" -to BUTTON[1]
+set_instance_assignment -name IO_STANDARD "3.3-V LVTTL" -to BUTTON[0]
```

Comment:
I think the "V SCHMITT TRIGGER" is a good thing..
Did you replace this for a reason?

Reply:
Taken care of.